src/wasm-traversal.h

```diff
@@ -176,17 +176,33 @@ struct Walker : public VisitorType {
     setFunction(func);
     static_cast<SubType*>(this)->doWalkFunction(func);
     static_cast<SubType*>(this)->visitFunction(func);
+    setFunction(nullptr);
```

Comment:
This seems right in general. Curious if there was behavior you were observing, or just noticing that this should be cleaned up while you were here.